openkfw/Oscar
# Add tile layer type and format to geodata schema

## Description

- geodata schema updated so we can use "tiles" in format and "tile" in geometryDataTypes in GeoData config. Also added notice to changelog, that tileDataUrl and tileAttributions are being deprecated and changed FE according to this. 

## Motivation and Context

- after mongoDB refactor to Postgresql, the filter in layers is changed(for postgre version) so now it expect all layers to have georeferenceId Point and Tile layers included, so geoData are needed for these layers now and we need format and layer type for Tile layer
- this change also affected other part of code, so now we can use 'apiUrl' key instead of 'tileDataUrl' key and 'metadata.description' key instead of 'tileAttributions' key so tileDataUrl and tileAttributions are being deprecated
- after merging this branch, create a new release if possible

##  How to test
- add metadata attributions prop to MapLayers and GeoData config, run app and initial load and see if you can find them in map info.

## Types of changes




## Checklist:
